build.gradle

```diff
@@ -58,9 +57,12 @@ versioneye {
 }
 
 allprojects {
+    apply plugin: 'java'
     apply plugin: 'jacoco'
     apply plugin: 'nebula.lint'
 
+    sourceCompatibility = 1.8
```

Comment:
Ya I don't like this being here.
But what I learned working on this is:
```groovy
sourceCompatibility = 1.8 // Does nothing
apply plugin: 'java'
```
Or
```groovy
sourceCompatibility = 1.8 // Does nothing
apply plugin: 'java'
sourceCompatibility = sourceCompatibility // Still does nothing
```
While
```groovy
apply plugin: 'java'
sourceCompatibility = 1.8 // Checks the Java version
```